.gitignore

```diff
@@ -5,3 +5,4 @@
 /tools
 /vendor
 /build
+/.phpunit.result.cache
```

Comment:
sorry, not related to this fix.... ignore phpunit cache generated during test suite run